File: #100/number15.php
<?php

$numbersSummary = 0;

$numbersCount = intval(trim(fgets(STDIN)));

for ($i = 0; $i < $numbersCount; $i++) {
	$inputNumber = intval(trim(fgets(STDIN)));

	if (!($inputNumber % 3) && $inputNumber < 300) {
		$numbersSummary += $inputNumber;
	}
}

echo($numbersSummary);
?>

File: #100/number15.cpp
#include <iostream>

using namespace std;

int main() {
	int numbersSummary = 0;
	int numbersCount, inputNumber;

	cin >> numbersCount;

	for (int i = 0; i < numbersCount; i++) {
		cin >> inputNumber;

		if (!(inputNumber % 3) && inputNumber < 300) {
			numbersSummary += inputNumber;
		}
	}
	cout << numbersSummary << endl;
}